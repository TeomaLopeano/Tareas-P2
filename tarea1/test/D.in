insertarAlInicio 1 2.1
insertarAlInicio 1 1.1
insertarAlInicio 11 11.1
insertarAlInicio 15 15.0
removerDeCadena 15
imprimirCadena
estaEnCadena 15
infoCadena 1
removerDeCadena 1
estaEnCadena 1
infoCadena 1
removerDeCadena 1
estaEnCadena 1
removerDeCadena 11
imprimirCadena
insertarAlInicio 1 3.1
estaEnCadena 1
infoCadena 1
imprimirCadena
Fin