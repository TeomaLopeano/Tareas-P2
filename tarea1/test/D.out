1>Se insertó (1,2.10) al inicio de la cadena.
2>Se insertó (1,1.10) al inicio de la cadena.
3>Se insertó (11,11.10) al inicio de la cadena.
4>Se insertó (15,15.00) al inicio de la cadena.
5>Se removió la primera aparición de 15 de la cadena.
6>(11,11.10)(1,1.10)(1,2.10)
7>15 no está en la cadena.
8>El elemento es (1,1.10)
9>Se removió la primera aparición de 1 de la cadena.
10>1 está en la cadena.
11>El elemento es (1,2.10)
12>Se removió la primera aparición de 1 de la cadena.
13>1 no está en la cadena.
14>Se removió la primera aparición de 11 de la cadena.
15>
16>Se insertó (1,3.10) al inicio de la cadena.
17>1 está en la cadena.
18>El elemento es (1,3.10)
19>(1,3.10)
20>Fin.
